docs/index.md

```diff
@@ -213,10 +208,8 @@ about:
   documentation: https://rich.readthedocs.io
   repository: https://github.com/Textualize/rich
 ```
-</details>
```

Comment:
I removed the `<details>` collapsible section tags because it wasn't displaying properly on the docs site (the repo's top-level `README.md` is fine, however). The monospace words (anything wrapped in a "`") weren't showing up as monospaced and also the filenames were off on their own and not showing up in bold. I tried editing this in multiple ways to keep it collapsible, but no matter what I tried it never seemed to render properly.

Please reference the screenshot below for what I mean:

<img width="847" alt="screenshot1" src="https://github.com/prefix-dev/rattler-build/assets/28930622/173a71e7-7fc5-48b6-8739-dff0956ae2a0">